docs/src/mathematical-formulation.md

```diff
@@ -0,0 +1,78 @@
+# [Mathematical Formulation](@id math-formulation)
+
+This section shows the mathematical formulation of the model.
+
+## [Sets](@id math-sets)
+
+Name|Description|Elements
+ ---|---|---:
+$\mathcal{A}$           | Energy assets                         | $a \in \mathcal{A}$
+$\mathcal{A_c}$         | Consumer energy assets                | $\mathcal{A_c} \subseteq \mathcal{A}$
+$\mathcal{A_p}$         | Producer energy assets                | $\mathcal{A_p} \subseteq \mathcal{A}$
+$\mathcal{A_i}$         | Energy assets with investment method  | $\mathcal{A_i} \subseteq \mathcal{A}$
+$\mathcal{A_b}$         | Energy assets with balance method     | $\mathcal{A_b} \subseteq \mathcal{A}$
+$\mathcal{RP}$          | Representative periods                | $rp \in \mathcal{RP}$
+$\mathcal{K}$           | Time steps within the $rp$            | $k \in \mathcal{K}$
+$\mathcal{F}$           | Flow connections between two assets   | $f \in \mathcal{F}$
+$\mathcal{F_{rec}}(a)$  | Set of flows with receiving asset $a$ | $\mathcal{F_{rec}}(a) \subseteq \mathcal{F}$
+$\mathcal{F_{snd}}(a)$  | Set of flows with sending asset $a$   | $\mathcal{F_{snd}}(a) \subseteq \mathcal{F}$
+
+## [Parameters](@id math-parameters)
+
+Name|Description|Units
+ ---|---|---
+$p^{investment\_cost}_{a}$ | Investment cost  of asset units      | [kEUR/MW/year]
+$p^{variable\_cost}_{a}$   | Variable cost of asset units         | [kEUR/MWh]
+$p^{unit\_capacity}_{a}$   | Capacity of asset units              | [MW]
+$p^{rp\_weight}_{rp}$      | Representative period weight         | [h]
+$p^{profile}_{a,rp, k}$    | Asset production/consumption profile | [p.u.]
+$p^{peak\_demand}_{a}$     | Peak demand                          | [MW]
+$p^{init\_capacity}_{a}$   | initial capacity of asset units      | [MW]
```

Comment:
Please check the superscripts, they are displayed wrong.

Reply:
I don't know if the problem is in the online preview that GitHub provides, since the preview in the editor for the markdown file seems correct.

Reply:
<img width="260" alt="image" src="https://github.com/TNO-Tulipa/TulipaBulb.jl/assets/12887482/ed84a431-27ab-49fe-8512-d66390cab151">

Reply:
The online preview of GitHub is possibly different. Markdown is very loose with that, and there is no specification for how to treat LaTeX.

Reply:
There is no easy way to fix, but also notice that the superscript if all math, not text. So you could change to something like `$p^{\text{unit capacity}}$`: $p^{\text{unit capacity}}$ Notice the lack of _.